### PR TITLE
Release time

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies
Get release time for current package being analyzed

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ bandit = "*"
 click = "*"
 thoth-common = "*"
 thoth-analyzer = "*"
+thoth-python = {editable = true, path = "/home/kpostlet/project/python"}
```
Comment (goern) [inline Pipfile]: This looks a little wrong

Comment (pacospace) [inline Pipfile]: should be thoth-python = "*"?

**Review by goern — CHANGES_REQUESTED** [review]

**Review by pacospace — COMMENTED** [review]
One comment, otherwise great!!

File: Pipfile (hunk 1 above)
Comment (KPostOffice) [reply Pipfile]: Oops, my bad. left over from local testing :/

File: thoth/si_bandit/version.py [hunk 2]
```diff
@@ -18,4 +18,4 @@
 """This file carries the version and title of the si-bandit project."""
 
 __title__ = "thoth-si-bandit"
-__version__ = "0.1.0"
+__version__ = "0.1.1"
```
Comment (harshad16) [inline thoth/si_bandit/version.py]: This would be taken care by kebechet on releases. May be explicit change is not needed here.

File: Pipfile [hunk 3]
```diff
@@ -10,7 +10,6 @@ bandit = "*"
 click = "*"
 thoth-common = "*"
 thoth-analyzer = "*"
-thoth-python = {editable = true, path = "/home/kpostlet/project/python"}
```
Comment (goern) [inline Pipfile]: removed completely?!

**Review by goern — CHANGES_REQUESTED** [review]

File: thoth/si_bandit/version.py (hunk 2 above)
Comment (goern) [reply thoth/si_bandit/version.py]: Oh ja right

**Pull request merged**